<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginTextEditor</class>
 <widget class="QStackedWidget" name="loginTextEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>434</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>3</number>
  </property>
  <widget class="QWidget" name="user_page">
   <widget class="QFrame" name="user_information_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="user_change_username_pushButton">
        <property name="text">
         <string>CHANGE USERNAME</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="user_change_password_pushButton">
        <property name="text">
         <string>CHANGE PASSWORD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="user_logout_pushButton">
        <property name="text">
         <string>LOGOUT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="user_file_frame">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>371</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>371</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="user_file_listWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="user_create_file_pushButton">
        <property name="text">
         <string>CREATE NEW FILE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="user_add_pushButton">
        <property name="text">
         <string>ADD FILE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="signup_page">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>371</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" alignment="Qt::AlignVCenter">
      <widget class="QLabel" name="signup_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;SIGNUP&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Surname</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="signup_surname_lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="signup_name_lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="signup_email_label">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="signup_email_lineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="signup_user_label">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="signup_username_lineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="signup_password_label">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="signup_password_lineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="signup_login_pushButton">
       <property name="text">
        <string>Already a member? Click to login</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="singup_register_pushButton">
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>SIGNUP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="login_page">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>60</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="login_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LOGIN&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="login_signin_pushButton">
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="login_password_label">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="login_password_lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="login_email_label">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="login_email_lineEdit"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="login_signup_pushButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">backgrount-color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Need Account?   | register</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="connect_page">
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>341</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QPushButton" name="connect_pushButton">
       <property name="text">
        <string>CONNECT</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="connect_address_label_2">
         <property name="text">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="connect_address_lineEdit_"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Insert a valid IP address to connect to a server</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
